packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -445,6 +475,7 @@ const checkAccountStatus = (
     const {
         authenticated,
         cashier_locked,
+        system_maintenance,
```

Comment:
system_maintenance is not coming from get_account_status.status, it came from get_account_status.cashier_validation array